test/parallel/test-cluster-bind-privileged-port.js

```diff
@@ -21,6 +21,18 @@
 
 'use strict';
 const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+const net = require('net');
+const { execSync } = require('child_process');
+
+if (common.isLinux) {
+  const sysctlOutput = execSync('sysctl net.ipv4.ip_unprivileged_port_start').toString();
```

Comment:
Add a comment?

Reply:
Sure, should I add one now? (PR is closing to merging I think...)

Reply:
> PR is closing to merging I think...

I am not sure what you mean here, you can still push changes and I'll re-run CI after :]

If you prefer to put it in another change to practice making changes in Node.js as a new contributor - that's also fine with me.

Reply:
I would like it if I could put it in another change so I can practice making changes, that would be really helpful!